packages/core/package.json

```diff
@@ -116,7 +117,7 @@
     "jest-matcher-specific-error": "^1.0.0",
     "jsonc-eslint-parser": "^2.4.0",
     "lint-staged": "^15.0.0",
-    "nock": "^13.3.1",
+    "nock": "^14.0.0-beta.5",
```

Comment:
nit: is this expected update?

Reply:
yes. `ky` uses `fetch`, only `nock` v14 supports it

Reply:
added to the pull request description